ecs-cli/modules/cli/compose/entity/entity_helper.go

```diff
@@ -232,24 +232,20 @@ func ConvertMapToSlice(mapItems map[string]bool) []*string {
 // GetTaskGroup returns an auto-generated formatted string
 // that can be supplied while starting an ECS task and is used to identify the owner of ECS Task
 func GetTaskGroup(entity ProjectEntity) string {
-	return composeutils.GetTaskGroup(getProjectPrefix(entity), GetProjectName(entity))
+	return composeutils.GetTaskGroup(GetProjectName(entity))
```

Comment:
This would also need to be ComposeProjectNamePrefix + projectName or `compose ps` with ini config won't work

Reply:
Good catch! I've updated it.